### PR TITLE
docs: update read profile data 

### PR DESCRIPTION
**Pull request by magalimorin18**

### CODE REVIEW

File: docs/learn/dapp-developer/read-profile-data.md [hunk 1]
```diff
@@ -7,115 +7,106 @@ description: Learn how to read profile data from your Universal Profile.
...
 ```
 
-## Get all profile data keys
+## Create erc725 instance
 
-:::tip
+:::info How is data stored on a Universal Profile
 
-🔍 You can inspect a profile smart contract (or any ERC725 contract) using 🔎 [ERC725 Inspect](https://erc725-inspect.lukso.tech/inspector) to see all its 🗂️ [ERC725Y data keys](../../standards/lsp-background/erc725#erc725y-generic-data-keyvalue-store).
+Each data on a Universal Profile is stored into a generic key-value [ERC725Y](https://github.com/ethereum/ercs/blob/master/ERCS/erc-725.md#erc725y) store which allows storing an unlimited amount of data in the smart contract.
```
Comment (Hugoo) [inline docs/learn/dapp-developer/read-profile-data.md:22]:
```suggestion
Each data on a Universal Profile is stored into a generic key-value [ERC725Y](https://docs.lukso.tech/standards/lsp-background/erc725#erc725y-generic-data-keyvalue-store) store which allows storing an unlimited amount of data in the smart contract.
```

NOTE: please use RELATIVE link and not absolute link as per suggestion above. This is just to give the example.

File: docs/learn/dapp-developer/read-profile-data.md [hunk 2]
```diff
@@ -7,115 +7,106 @@ description: Learn how to read profile data from your Universal Profile.
...
-:::
+ERC725Y standardizes a mapping of data keys to data values in order to have the ability to add or remove data across time without the need of redeploying the contract. It gives flexibility to the contract storage. On the smart contract, data keys are represented as bytes32 values and data values under these keys are stored as bytes.
 
-<details>
-<summary>
-<a href="../../standards/universal-profile/lsp3-profile-metadata">LSP3 - Profile Metadata</a> describes the data in the Universal Profile contract's <a href="../../standards/lsp-background/erc725#erc725y-generic-data-keyvalue-store">ERC725Y data storage</a>. You can get the content of these data keys directly using the ⚒️ <a href="../../tools/erc725js/classes/ERC725#getdata"> erc725.js</a>  library. 👇
-</summary>
-
-<div>
+LSP2 - ERC725YJSONSchema is the standard used by the LSPs to organize how the data should be represented as bytes32/bytes pairs.
```
Comment (Hugoo) [inline docs/learn/dapp-developer/read-profile-data.md:26]: maybe you can link to:

https://docs.lukso.tech/standards/generic-standards/lsp2-json-schema

File: docs/learn/dapp-developer/read-profile-data.md [hunk 3]
```diff
@@ -7,115 +7,106 @@ description: Learn how to read profile data from your Universal Profile.
...
-:::tip Universal Profile Explorer
-
-The explorer on 🔮 [universalprofile.cloud](https://universalprofile.cloud/) indexes all deployed Universal Profiles on the LUKSO network. You can try out the following examples with any Universal Profile address.
-
-:::
-
-## Setup
-
-To easily interact with a profile you should use the ⚒️ [erc725.js](https://npmjs.com/package/@erc725/erc725.js) library. It is able to fetch and encode profile and contract data easily. You can install it in your project using:
+## Install [erc725.js](https://npmjs.com/package/@erc725/erc725.js) library
```
Comment (Hugoo) [inline docs/learn/dapp-developer/read-profile-data.md:12]: There is also content here:

https://docs.lukso.tech/tools/erc725js/getting-started

Let's make sure we don't duplicate content from that side also

File: docs/learn/dapp-developer/read-profile-data.md [hunk 4]
```diff
@@ -144,154 +134,197 @@ You can give it a try with this profile address: [`<myProfileAddress> = 0xE1F684
...
 
-:::
+Example to retrieve the content of the JSON file from the verifiable URI stored on the smart contract. It will download the JSON file and verify its hash automatically.
 
-## Fetch Assets and Universal Receiver
+<details>
+    <summary>Show code snippet</summary>
 
-Instead of using the [`LSP3Profile`](../../standards/universal-profile/lsp3-profile-metadata) key, you can also use other data keys like [`LSP12IssuedAssets[]`](../../standards/universal-profile/lsp12-issued-assets), [`LSP5ReceivedAssets[]`](../../standards/universal-profile/lsp5-received-assets), or [`LSP1UniversalReceiverDelegate`](../../standards/generic-standards/lsp1-universal-receiver-delegate), as described in the following example:
+```js
```
Comment (Hugoo) [inline docs/learn/dapp-developer/read-profile-data.md:227]: does the syntax highlighter supports `ts` ? The code below is TS so might be more relvant to put ts instead if it is supported

File: docs/learn/dapp-developer/read-profile-data.md [hunk 5]
```diff
@@ -144,154 +134,197 @@ You can give it a try with this profile address: [`<myProfileAddress> = 0xE1F684
...
+    ],
+  },
+];
 ```
 
 </details>
+
+## Take advantage of our ERC725 inspect tool
+
+🔍 You can retrieve encoded Universal Profile data easily with our [ERC725 Inspect](https://erc725-inspect.lukso.tech/inspector) tool. You only have to paste your Universal Profile address and choose the data key to retrieve data from.
```
Comment (Hugoo) [inline docs/learn/dapp-developer/read-profile-data.md:330]: cool!

**Review by Hugoo — APPROVED** [review]
lgtm - note i didnt run the code

File: docs/learn/dapp-developer/read-profile-data.md (hunk 2 above)
Comment (CJ42) [reply docs/learn/dapp-developer/read-profile-data.md:26]:
```suggestion
[LSP2 - ERC725YJSONSchema](../../../[standards/generic-standards/lsp2-json-schema](https://docs.lukso.tech/standards/generic-standards/lsp2-json-schema)) is the standard used by the LSPs to organize how the data should be represented as bytes32/bytes pairs.
```

**Pull request merged**